nixos/modules/services/security/physlock.nix

```diff
@@ -49,6 +49,22 @@ in
         '';
       };
 
+      displayIssue = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Whether to display /etc/issue when locked with physlock.
+        '';
+      };
+
+      displayUser = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to display the currently logged in user when locked with physlock.
+        '';
+      };
```

Comment:
you might actually want to do a changelog entry where you say that the module options where added and also mention that the upstream was changed to a fork because the original one was archived.